pom.xml

```diff
@@ -194,11 +227,62 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
+1 to have this dependency-plugin do this, as it makes setting up the project a lot easier. very good idea.

However, this shouldn't happen in the openshift build on openshift online, as that's just going to slow down the build a lot.
So we don't want this to happen if `-Popenshift` is active.